src/infra/service/graph.js

```diff
@@ -7,18 +7,21 @@ const get = (url: string): Promise<any> => new Promise((resolve, reject) => {
   });
 });
 
+const getWithToken = (url: string, accessToken: string): Promise<any> => new Promise((resolve, reject) => {
```

Comment:
This can use existing `get`, because `window.FB` is actually save token from our current authentication.